src/app/notebooks-warning/notebooks-warning.component.html

```diff
@@ -0,0 +1,5 @@
+<mat-card class="alert-info mat-elevation-z w-100" data-cy="notebooks-warning">
```

Comment:
This banner is good, just putting it out there that I had the brief thought about whether this should be an `alert-warning` but that might look too concerning instead of informational :slightly_smiling_face:

Reply:
We definitely considered `alert-warning`, but I agree it may be a bit too concerning looking ⚠️